scripts/lb-wrapper

```diff
@@ -40,21 +36,26 @@ fi
 #     exit 1
 # fi
 
-if ! command -v "${XKLB_EXECUTABLE}"; then
-    log "Error" "xklb could not be found. Please install xklb and try again."
+if [ "$command_type" == "tubeadd" ]; then
+    XKLB_CMD="${XKLB_EXECUTABLE} tubeadd ${XKLB_DB_FILE} ${URL} ${VERBOSITY}"
+elif [ "$command_type" == "dl" ]; then
+    FORMAT_OPTIONS="--format best --format-sort 'tbr~1000'"
```

Comment:
Special Request: Please keep all "CONSTANTS" near the top of the bash script (this one especially should stay where it is around Line 13, with the English explanation about alternatives for OOM debugging etc!)

```
FORMAT_OPTIONS="--format best --format-sort 'tbr~1000'"
```

Reply:
Done